hack/update-yamls-with-the-same-content.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -e
```

Comment:
`update-yamls-with-the-same-content.sh` is long. how about:
`sync-common-yaml-contents.sh`